### PR TITLE
feat: display effective status, and state in 'status'

### PR DESCRIPTION
**Pull request by Dav-14**

### CODE REVIEW

File: components/fctl/membership-swagger.yaml [hunk 1]
```diff
@@ -197,15 +204,9 @@ paths:
         schema:
           type: string
         required: true
-      - name: deleted
```
Comment (gfyrag) [inline components/fctl/membership-swagger.yaml:200]: Doing this case a breaking change for fctl

Comment (Dav-14) [reply components/fctl/membership-swagger.yaml:200]: same for `--disabled` 


1/ Do I mark them deprecated ?
2/ Do I keep those query paramaters membership side ? (Map it on --all)

File: components/fctl/cmd/stack/show.go [hunk 2]
```diff
@@ -107,11 +108,18 @@ func (c *StackShowController) Run(cmd *cobra.Command, args []string) (fctl.Rende
 
 	stackClient, err := fctl.NewStackClient(cmd, cfg, stack)
 	if err != nil {
+		if strings.Contains(err.Error(), "404 page not found") && stack.Status != "ACTIVE" {
```
Comment (gfyrag) [inline components/fctl/cmd/stack/show.go:111]: Beurk :)

**Review by gfyrag — APPROVED** [review]

**Review by gfyrag — APPROVED** [review]
Fix the 404 please ;)

**Pull request merged**